package_locator/common.py

```diff
@@ -69,7 +69,7 @@ def get_base_repo_url(repo_url):
 
 def search_for_github_repo(data):
     urls = set()
-    url_pattern = re.compile(r"""https?:\/\/(?:www\.)?github\.com[^\s)|<|>]+""")
+    url_pattern = re.compile(r"""https?:\/\/(?:www\.)?github\.com[^\s)|<|>\"]+""")
```

Comment:
Can you show me a test case where a backslash would help? 
Also, are we sure that a backslash cannot legitimately appear within an API?